challenge_0/cpp/sven/src/HelloWorld.cpp

```diff
@@ -0,0 +1,7 @@
+#include <iostream>
+
+int main()
+{
+	std::cout << "Hello World!" << std::endl;
```

Comment:
I *really* appreciate your use of `std::` and not `using namespace std;`. Same goes for `std::endl`.